components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -226,7 +226,7 @@ spec:
             value: "/mnt/tyk-gateway/policies/policies.json"
         {{- end}}
           - name: TYK_GW_HASHKEYS
-            value: "true"
+            value: {{ .Values.global.hashKeys | toString | default "true" | quote }}
```

Comment:
No need to set the default value here. It should be enough if we set it to true in default values.yaml